translations/strings.yaml

```diff
@@ -220,6 +220,10 @@ whatsNewReleaseNotes:
   tourSystemNotifHeader: What’s new in VPN
   tourSystemNotifBody: Check out all the new features available in the latest version of Mozilla VPN!
   tourSystemNotifCTA: Take the tour
+  # Multi-account containers
+  multiAccountContainersHeader: Mozilla VPN in Firefox
+  multiAccountContainersShortDescription: Mozilla VPN is now integrated with the Multi-Account Containers add-on for more privacy and security.
```

Comment:
Please add a note that `Multi-Account Containers` is the name of the add-on, and shouldn't be translated.